### PR TITLE
Correct the English a little bit

### PR DESCRIPTION
**Pull request by irfanb**

### CODE REVIEW

File: docs/manual/client-side-caching.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ The Redis client-side caching support is called _Tracking_, and has two modes:
 * In the _broadcasting_ mode, the server does not attempt to remember what keys a given client accessed, so this mode costs no memory at all in the server side. Instead clients subscribe to key prefixes such as `object:` or `user:`, and receive a notification message every time a key matching a subscribed prefix is touched.
 
 To recap, for now let's forget for a moment about the broadcasting mode, to
-focus on the first mode. We'll describe broadcasting later more in details.
+focus on the first mode. We'll describe broadcasting later in more detail.
```
Comment (itamarhaber) [inline docs/manual/client-side-caching.md:96]: Even better?

```suggestion
focus on the first mode. We'll describe broadcasting in more detail later.
```

**Comment by itamarhaber** [comment]
Hi @irfanb - thanks for reading the docs and spotting glitches!

**Comment by irfanb** [comment]
Hiya - I've adopted your proposed change. I am not sure who is responsible for merging this PR.

**Review by itamarhaber — APPROVED** [review]
